package.json

```diff
@@ -56,6 +56,56 @@
     "onStartupFinished"
   ],
   "main": "./out/extension.js",
+  "runme": {
+    "features": [
+      {
+        "name": "ForceLogin",
```

Comment:
Let's turn this array with a `name` per item into a object that uses the name as key. Something like this:

```
  "runme": {
    "features": {
      "forceLogin": {
        "enable": true,
        "conditions": {
          "os": "All",
          "vsCodeVersion": ">=1.58.0",
          "allowedExtensions": [
            "stateful.platform"
          ]
        }
      },
      "escalate": {
        "enable": true,
        "conditions": {
          "os": "All",
          "vsCodeVersion": ">=1.58.0",
          "statefulAuthRequired": true,
          "allowedExtensions": [
            "stateful.platform"
          ]
        }
      },
      "share": {
        "enable": true,
        "conditions": {
          "os": "All",
          "vsCodeVersion": ">=1.58.0",
          "allowedExtensions": [
            "stateful.platform",
            "stateful.runme"
          ]
        }
      },
      "gist": {
        "enabled": true,
        "conditions": {
          "os": "All",
          "vsCodeVersion": ">=1.58.0",
          "allowedExtensions": [
            "stateful.runme"
          ]
        }
      }
    }
  },
```